src/pages/HomePage.tsx

```diff
@@ -1,27 +1,29 @@
-import { Heading } from "@chakra-ui/react";
 import React from "react";
-import { User, usersApi } from "@/utils/fetcher";
+import { Heading, Text } from "@chakra-ui/react";
+import { Movie, moviesApi } from "@/utils/fetcher";
 
 const HomePage = () => {
   const [loading, setLoading] = React.useState(false);
-  const [users, setUsers] = React.useState<User[]>([]);
+  const [movies, setMovies] = React.useState<Movie[]>();
```

Comment:
ini butuh dikasih default value

```suggestion
  const [movies, setMovies] = React.useState<Movie[]>([]);
```